azure-pipelines/doc-build.yml

```diff
@@ -0,0 +1,99 @@
+## Specify which branches you want to trigger for continous deployment and/or applicable for pull requests.
+## Otherwise it triggers all of them.
+trigger:
+- master
+
+pr:
+- master
+
+# Variable we can refernece later
+variables:
+  system.debug: 'true'
+  ref.data.home: '$(Agent.BuildDirectory)/tardis-refdata'
+
+  ref.data.github.url: 'https://github.com/tardis-sn/tardis-refdata.git'
+  tardis.build.dir: $(Build.Repository.LocalPath)
+
+  travis.repo.slug: 'tardis-sn/tardis'
+
+
+jobs:
+
+- job: 'Test'
+  pool:
+    vmImage: $[variables.vm_Image]
+
+#### Basic setup of our virtual machine(vm)
+  strategy:
+    matrix:
+      linux:
+        vm_Image: 'Ubuntu-16.04'
+        conda: '/usr/share/miniconda'
+      mac:
+        vm_Image: 'macOS-10.13'
```

Comment:
i don't think we need to build this twice, right?

Reply:
I agree!